manager/state/raft/raft.go

```diff
@@ -270,6 +275,9 @@ func (n *Node) JoinAndStart() error {
 			n.Node = raft.StartNode(n.Config, []raft.Peer{})
 
 			if err := n.registerNodes(resp.Members); err != nil {
```

Comment:
Should this be `=` rather than `:=`? (as in, are we creating a local `err` rather than using the function one which won't get caught by the defer)?